modules/dxp/apps/analytics/analytics-reports-journal-impl/src/main/java/com/liferay/analytics/reports/journal/internal/info/item/JournalArticleAnalyticsReportsInfoItem.java

```diff
@@ -171,8 +170,10 @@ private User _getUser(JournalArticle journalArticle) {
 	@Reference
 	private JournalArticleLocalService _journalArticleLocalService;
 
-	@Reference
-	private LayoutDisplayPageObjectProviderAnalyticsReportsInfoItem
+	@Reference(
+		target = "(component.name=com.liferay.analytics.reports.layout.display.page.internal.info.item.LayoutDisplayPageObjectProviderAnalyticsReportsInfoItem)"
```

Comment:
Hey @MrJohn1911, please add a property to the service and use it in the filter instead of the `component.name`. Thx